aps/toolbox/copy_rms_param_to_ertbox_grid.py

```diff
@@ -196,7 +195,7 @@ def from_geogrid_to_ertbox(project, params, seed=12345):
             "Use 'Eclipse grid standard' (upper left corner) as "
             'common grid index origin (right-handed grid) in FMU projects using ERT.'
         )
-    if ertboxgrid_handedness != roxar.Direction.right:
+    if ertboxgrid_handedness != rmsapi.Direction.right:
```

Comment:
We may want to import `Direction`, so that we can do the import logic consistently in one place